controller/libpostal.js

```diff
@@ -0,0 +1,30 @@
+const text_analyzer = require('pelias-text-analyzer');
+const _ = require('lodash');
+const iso3166 = require('iso3166-1');
+
+function setup(should_execute) {
+  function controller( req, res, next ){
+    // bail early if req/res don't pass conditions for execution
+    if (!should_execute(req, res)) {
+      return next();
+    }
+
+    // parse text with query parser
+    const parsed_text = text_analyzer.parse(req.clean.text);
+
+    if (!_.isUndefined(parsed_text)) {
```

Comment:
I was on a lodash roll!  Seriously, though, it's probably overkill for this particular conditional since it only ever returns an object or undefined, but I'm pretty sure that this a helper function so that people don't do `if (!parsed_text) { ... }` where the intent was only to check for `undefined` instead of all falsey values.